SalesforceLiveAgentApp.ts

```diff
@@ -19,6 +19,7 @@ import { AppSettings } from './config/AppSettings';
 import { AvailabilityEndpoint } from './endpoints/AvailabilityEndpoint';
 import { HandoverEndpoint } from './endpoints/HandoverEndpoint';
 import { DialogflowAgentAssignedClass } from './lib/DialogflowAgentAssignedHandler';
+import { IdleSessionTimeoutProcessor } from './lib/IdleSessionTimeoutProcessor';
```

Comment:
I think you forgot to add `IdleSessionTimeoutProcessor` file in the `lib` folder 😜